.travis-ci.sh

```diff
@@ -75,6 +75,16 @@ function build_one {
     echo "End   availability check."
     case $TRAVIS_OS_NAME in
     linux)
+      # we need fresh gcc and binutils, maybe...
+      if [ `opam install --dry-run $pkg | grep -c mirage-entropy-xen` -gt 0 ] ; then
```

Comment:
while the gcc piece is rather simple, the binutils is a bit more tricky since `apt-get` won't install fresh binutils on travis (due to release version mismatch)... this is why there's a manual download and a forced installation with `dpkg`...